src/device-registry/utils/create-event.js

```diff
@@ -589,6 +591,16 @@ const createEvent = {
                 filter,
                 page,
               });
+              if (language !== undefined && constants.ENVIRONMENT === "STAGING ENVIRONMENT") {
+                let data = responseFromListEvents.data[0].data;
+                for (const event of data) {
```

Comment:
Hi @BenjaminSsempala , this logic is still confusing me a little bit :)......
Especially the part of manipulating a const variable....and the logical flow of things is bit confusing!
Can we do this translation from somewhere else within the create-health-tips use case and minimise changes to this particular util function?

Reply:
@Baalmart The translation is also in the create-health-tips, however The tips in the mobile app come along with the measurements using the `/events` endpoint.